generator/mavgen_cs.py

```diff
@@ -341,7 +341,9 @@ def generate(basename, xml):
     #print("Cmd:" + compileCommand)
     res = os.system (compileCommand)
     
-    if res == '0':
-        print("Generated %s OK" % filename)
+    if res == 0:
+        print("Generated %s OK" % outputLibraryPath)
     else:
         print("Error")
+        print("Error: Compilation failed. (" + str(res) + ")")
```

Comment:
Weird... somehow that commit got mangled.  Originally the more detailed error message replaced the one above it.  Thanks for catching that!